### DISCUSSION TITLE
Config files for *all* default settings & presets

### DISCUSSION THREAD

**Discussion by notDavid** [jmbannon/ytdl-sub#979] [Q&A] [answered]
Hi there,

i'm reading the Docs and trying to wrap my head around all the settings... and was wondering;

Is it possible to generate / download the "default config files" which contain **all** the default settings and presets that are used by default?

Thank you!

**Comment by jmbannon** [comment, marked as answer]
Hi @notDavid , you can find them all here: https://github.com/jmbannon/ytdl-sub/tree/master/src/ytdl_sub/prebuilt_presets

In addition,.you can run with --log-level debug and it will print all settings used in a subscription
